foosballv2s/Resources/values/Strings.xml

```diff
@@ -7,7 +7,7 @@
     <string name="first_team_name">First team name</string>
     <string name="second_team_name">Second team name</string>
     <string name="submit">Submit</string>
-    <string name="prev">Previous Settings</string>
+    <string name="prev">Previous Color</string>
```

Comment:
Translate this into lithuanian in values-lt/Strings.xml

Reply:
Done, will push soon